magenta-lib/src/main/scala/magenta/tasks/UpdateCloudFormationTask.scala

```diff
@@ -8,6 +8,13 @@ import com.amazonaws.services.cloudformation.model._
 import scalax.file.Path
 import collection.convert.wrapAsScala._
 
+object UpdateCloudFormationTask {
+  trait ParameterValue
+  case class StringValue(value: String) extends ParameterValue
+  case class UseExistingValue() extends ParameterValue
```

Comment:
I'd make this a `case object` and the trait `sealed` so that the compiler can do exhaustivity checking.